CONTRIBUTING.md

```diff
@@ -24,6 +24,16 @@ A class should only declare one specifier per specifier type.
 All non-private members need to be documented using Doxygen-style comments.
 For functions and methods, parameters are documented first and return type next.
 
+### Clang
+We use clang-format (v 3.9) to enforce our coding standard.
+To format a specific file, use clang-format -i PATH_TO_FILE/FILENAME when in the /src directory.
+
+Clang-format for windows can be grabbed from here: 
+http://releases.llvm.org/
+
+The options we use can be found here:
```

Comment:
We don't use all these options, though. A clearer explanation may be: "The options we use are specified in `src/.clang-format`. Explanations of the options can be found here: